src/components/containers/visibility.tsx

```diff
@@ -78,3 +79,19 @@ export const Mobile = ({
         <MobileLayer breakpoint={breakpoint_size}>{children}</MobileLayer>
     )
 }
+
+type TVisiblityProps = {
+    config: Partial<TVisiblity>
+    children?: ReactNode
+}
+
+export const Visibility = ({ children, config }: TVisiblityProps) => {
```

Comment:
@michio-rework If I got it correctly this component is going to replace other components in this file such as `Mobile` and `Desktop`, If that’s the case then I would suggest marking those components as deprecated to warn developers when they are using them in new places and inform them about the new component.

Reply:
yeah we will, it's gonna be another task. 
making these components deprecated will cause confusion right now. 
will do it in the next step ( iteration ).